pkg/controller/init.go

```diff
@@ -796,3 +796,30 @@ func (c *Controller) initDeleteOverlayPodsStaticRoutes() error {
 
 	return nil
 }
+
+func (c *Controller) initNodeChassis() error {
```

Comment:
what if the chassis id of a node is deleted, and then this node is rejoined in the cluster?
In this scenario, the func `InitChassisNodeTag` will not be invoked.
Also, the chassisGC  will not re-build the chassis annotation of node.

Reply:
> what if the chassis id of a node is deleted, and then this node is rejoined in the cluster? In this scenario, the func `InitChassisNodeTag` will not be invoked. Also, the chassisGC will not re-build the chassis annotation of node.

It is only when the OVS DB is deleted and reconstructed that the inconsistency with the node annotation will occur. In this case, I think we should rebuild the OVN-CNI-SERVER pod on the node to refresh the node annotations.
And, the chassis external_id tag will be written the next time the AddNode event comes.